qiskit/version.py

```diff
@@ -86,67 +88,28 @@ def get_version_info():
 
 
 class QiskitVersion(Mapping):
-    """A lazy loading wrapper to get qiskit versions."""
+    """DEPRECATED in 0.25.0 use qiskit.__version__"""
 
     __slots__ = ["_version_dict", "_loaded"]
 
     def __init__(self):
+        warnings.warn(
+            "qiskit.__qiskit_version__ is deprecated since "
+            "Qiskit Terra 0.25.0, and will be removed 3 months or more later. "
+            "Instead, you should use qiskit.__version__. The other packages listed in "
+            "former qiskit.__qiskit_version__ have their own __version__ module level dunder, "
+            "as standard in PEP 8.",
+            category=DeprecationWarning,
+        )
         self._version_dict = {
             "qiskit-terra": __version__,
-            "qiskit-aer": None,
-            "qiskit-ignis": None,
-            "qiskit-ibmq-provider": None,
             "qiskit": None,
         }
         self._loaded = False
 
     def _load_versions(self):
         from importlib.metadata import version
 
-        try:
-            # TODO: Update to use qiskit_aer instead when we remove the
-            # namespace redirect
-            from qiskit.providers import aer
-
-            self._version_dict["qiskit-aer"] = aer.__version__
-        except Exception:
-            self._version_dict["qiskit-aer"] = None
-        try:
-            from qiskit import ignis
-
-            self._version_dict["qiskit-ignis"] = ignis.__version__
-        except Exception:
-            self._version_dict["qiskit-ignis"] = None
-        try:
-            from qiskit.providers import ibmq
-
-            self._version_dict["qiskit-ibmq-provider"] = ibmq.__version__
-        except Exception:
-            self._version_dict["qiskit-ibmq-provider"] = None
-        try:
-            import qiskit_nature
-
-            self._version_dict["qiskit-nature"] = qiskit_nature.__version__
-        except Exception:
-            self._version_dict["qiskit-nature"] = None
-        try:
-            import qiskit_finance
-
-            self._version_dict["qiskit-finance"] = qiskit_finance.__version__
-        except Exception:
-            self._version_dict["qiskit-finance"] = None
-        try:
-            import qiskit_optimization
-
-            self._version_dict["qiskit-optimization"] = qiskit_optimization.__version__
-        except Exception:
-            self._version_dict["qiskit-optimization"] = None
-        try:
-            import qiskit_machine_learning
-
-            self._version_dict["qiskit-machine-learning"] = qiskit_machine_learning.__version__
-        except Exception:
-            self._version_dict["qiskit-machine-learning"] = None
```

Comment:
I'm actually debating whether we actually want to do this here. I'm thinking since we're just deprecating the `qiskit.__qiskit_version__` attribute we should just keep this working as is until we remove it.

Reply:
I just went ahead and did this in the interest of merging this in time for 0.25.0